tests/iana_registries/bootp_dhcp_parameters.rflx

```diff
@@ -0,0 +1,1101 @@
+       --
+       Nds_Servers => 85,
+
+       -- length = N, description = Novell Directory Services, Ref: rfc2241
+       --
+       Nds_Tree_Name => 86,
+
+       -- length = N, description = Novell Directory Services, Ref: rfc2241
+       --
+       Nds_Context => 87,
+
+       -- Ref: rfc4280
+       --
+       Bcmcs_Controller_Domain_Name_List => 88,
+
+       -- Ref: rfc4280
+       --
+       Bcmcs_Controller_Ipv4_Address_Option => 89,
+
+       -- length = N, description = Authentication, Ref: rfc3118
+       --
+       Authentication => 90,
+
+       -- Ref: rfc4388
+       --
+       Client_Last_Transaction_Time_Option => 91,
+
+       -- Ref: rfc4388
+       --
+       Associated_Ip_Option => 92,
+
+       -- length = N, description = Client System Architecture, Ref: rfc4578
+       --
+       Client_System => 93,
+
+       -- length = N, description = Client Network Device Interface, Ref: rfc4578
+       --
+       Client_Ndi => 94,
+
+       -- length = N, description = Lightweight Directory Access Protocol, Ref: rfc3679
+       --
+       Ldap => 95,
+
+       -- length = N, description = UUID/GUID-based Client Identifier, Ref: rfc4578
+       --
+       Uuid_Guid => 97,
+
+       -- length = N, description = Open Group's User Authentication, Ref: rfc2485
+       --
+       User_Auth_98 => 98,
+
+       -- Ref: rfc4776
+       --
+       Geoconf_Civic => 99,
+
+       -- length = N, description = IEEE 1003.1 TZ String, Ref: rfc4833
+       --
+       Pcode => 100,
+
+       -- length = N, description = Reference to the TZ Database, Ref: rfc4833
+       --
+       Tcode => 101,
+
+       -- length = 4, description = Number of seconds that DHCPv4 should be disabled, Ref:
+       --  rfc8925
+       --
+       Ipv6_Only_Preferred => 108,
+
+       -- length = 16, description = DHCPv4 over DHCPv6 Softwire Source Address Option, Re
+       -- f: rfc8539
+       --
+       Option_Dhcp4o6_S46_Saddr => 109,
+
+       -- length = N, description = NetInfo Parent Server Address, Ref: rfc3679
+       --
+       Netinfo_Address => 112,
+
+       -- length = N, description = NetInfo Parent Server Tag, Ref: rfc3679
+       --
+       Netinfo_Tag => 113,
+
+       -- length = N, description = DHCP Captive-Portal, Ref: rfc8910
+       --
+       Dhcp_Captive_Portal => 114,
+
+       -- length = N, description = DHCP Auto-Configuration, Ref: rfc2563
+       --
+       Auto_Config => 116,
+
+       -- length = N, description = Name Service Search, Ref: rfc2937
+       --
+       Name_Service_Search => 117,
+
+       -- length = 4, description = Subnet Selection Option, Ref: rfc3011
+       --
+       Subnet_Selection_Option => 118,
+
+       -- length = N, description = DNS domain search list, Ref: rfc3397
+       --
+       Domain_Search_119 => 119,
+
+       -- length = N, description = SIP Servers DHCP Option, Ref: rfc3361
+       --
+       Sip_Servers_Dhcp_Option => 120,
```

Comment:
The changed casing of abbreviations like `TLS` to `Tls` or `DHCP` to `Dhcp` is not that nice. Another approach to preserve the casing could be the following:
1. Replace unallowed characters by underscores (as it is already done).
2. Change first character of identifier and each character after an underscore to upper case, and leave all other characters unchanged.

Some examples:

- `SIP Servers DHCP Option` -> `SIP_Servers_DHCP_Option`
- `IEEE 802.1Q VLAN ID` -> `IEEE_802_1Q_VLAN_ID`
- `OPTION_V4_DOTS_RI` -> `OPTION_V4_DOTS_RI`
- `status-code` -> `Status_Code`
- `Vertical datum WGS 84 defined by EPSG CRS Code 4327` -> `Vertical_Datum_WGS_84_Defined_By_EPSG_CRS_Code_4327`

Reply:
Done in a092536